src/city/view.c

```diff
@@ -196,6 +196,9 @@ void city_view_set_camera(int x, int y)
 
 void city_view_set_camera_from_pixel_position(int x, int y)
 {
+    x = x < 0 ? 0 : x;
+    y = y < 0 ? 0 : y;
```

Comment:
Just wondering: why do we need these corrections?

Reply:
Otherwise when you scroll towards the very top it keeps moving up and jumping down frequently.